di_website/templates/news/news_story_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends "layouts/sidebar.html" %}
+
+{% load navigation_tags static wagtailcore_tags wagtailimages_tags card_tags %}
+
+{% block hero %}
+    {% include 'includes/heroes/hero-news.html' %}
+{% endblock hero %}
+
+{% block main %}
+    <section class="section section--post">
+        <div class="row row--narrow">
+            <div class="max-meter max-meter--center">
+                <div class="is-typeset is-typeset--additionals wysiwyg">
+                    {% for block in page.body %}
+                        {% if block.block_type == 'paragraph_block' or block.block_type == 'block_quote' %}
+                            {% include_block block %}
+                        {% endif %}
+                    {% endfor %}
+                </div>
+            </div>
+        </div>
+    </section>
+{% endblock %}
+
+<!-- TODO: include sidebar links from what we do once they're approved -->
+
+{% block after_content %}
+    {% if page.related_links %}
+        <section class="section">
+            <div class="row row--narrow">
+                <h2 class="section__heading">Related content</h2>
+                <div class="l-2up-3up">
+                    {% for related_page in related_pages %}
+                        {% with item=related_page.specific %}
```

Comment:
Could you please add a {% if item.pk != self.pk %} check or something of the like? I've got one news page and it's linking to itself as a related page.

Reply:
Ahh, great catch... thanks